src/project/internal/projectactionscontroller.cpp

```diff
@@ -1540,6 +1540,7 @@ io::path_t ProjectActionsController::selectScoreOpeningFile()
                                       trc("project", "Overture / Score Writer files (experimental)") + " (*.ove *.scw)",
                                       trc("project", "Bagpipe Music Writer files (experimental)") + " (*.bmw *.bww)",
                                       trc("project", "Guitar Pro files") + " (*.gtp *.gp3 *.gp4 *.gp5 *.gpx *.gp)",
+                                      trc("project", "MEI files") + " (*.mei)",
                                       trc("project", "Power Tab Editor files (experimental)") + " (*.ptb)",
```

Comment:
I believe Guitar Pro files and ptb files are somewhat related, so maybe better put MEI below these two rather than between them

```suggestion
                                      trc("project", "Power Tab Editor files (experimental)") + " (*.ptb)",
                                      trc("project", "MEI files") + " (*.mei)",
```